docs/source/api.rst

```diff
@@ -727,7 +727,8 @@ Property                                     Type     Units
 ``heatCapacityBtuPerHour``                   float    Btu/hr                      >0              no                              autosized by modeling engine if undefined
 ``energyFactor`` or ``uniformEnergyFactor``  float    fraction                    <1              no        BSA
 ``hotWaterTemperature``                      float    F                           >0              no        125 
-===========================================  =======  ==========================  ==============  ========  ====================  ============================================== 
+``performanceAdjustment``                    float    fraction                    see [#]_        no        see [#]_              Multiplier on efficiency, typically to account for cycling
```

Comment:
@kellyatradiantlabs Is this decimal number greater or equal or strictly greater than zero?

Reply:
Good question. OpenStudio does not specify a constraint on this value in their documentation, but I think we should implement a 0-1 constraint. I'll revise in our docs accordingly.